yb-voyager/cmd/exportData.go

```diff
@@ -111,14 +111,14 @@ func exportDataOffline() bool {
 	} else {
 		tableList = source.DB().GetAllTableNames()
 		finalTableList = sqlname.SetDifference(tableList, excludeTableList)
-		fmt.Printf("Num tables to export: %d\n", len(finalTableList))
-		utils.PrintAndLog("table list for data export: %v", finalTableList)
+		log.Infof("initial all tables table list for data export: %v", tableList)
+
+		finalTableList = source.DB().FilterEmptyTables(finalTableList)
```

Comment:
The `FilterEmptyTables()` and `FilterUnsupportedTables()` should return two values: `tableList` _after_ removing empty/unsupported tables and the `skippedTableList` for the list of empty/unsupported tables.

We should output on the screen messages like `Skipping empty tables [table1, table2, ...]` and `Skipping unsupported tables [table1, table2, ...]`.